application/comit_node/src/http_api/swap_resource.rs

```diff
@@ -64,12 +64,12 @@ pub enum IncludeState {
     No,
 }
 
-pub fn new_rfc003_hal_swap_resource<S: StateStore>(
+pub fn new_rfc003_siren_entity<S: StateStore>(
```

Comment:
:wrench: I think I missed the PR that restructured this stuff but a better name for this would be `retrieve_rfc003_siren_entity`.
:question: Can't we just make From implementation for (alice/bob)::State -> siren::Entity